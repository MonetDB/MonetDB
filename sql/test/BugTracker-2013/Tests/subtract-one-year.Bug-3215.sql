select cast ('2013-01-01' as date) + interval '-1' year;
