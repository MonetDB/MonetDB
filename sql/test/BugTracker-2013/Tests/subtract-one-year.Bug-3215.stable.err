stderr of test 'subtract-one-year.Bug-3215` in directory 'sql/test/BugTracker-2013` itself:


# 10:20:45 >  
# 10:20:45 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "gdk_dbfarm=/ufs/sjoerd/Monet-stable/var/MonetDB" "--set" "mapi_open=true" "--set" "mapi_port=31449" "--set" "mapi_usock=/var/tmp/mtest-17694/.s.monetdb.31449" "--set" "monet_prompt=" "--trace" "--forcemito" "--set" "mal_listing=2" "--dbname=mTests_test_BugTracker-2013" "--set" "mal_listing=0"
# 10:20:45 >  

# builtin opt 	gdk_dbname = demo
# builtin opt 	gdk_dbfarm = /ufs/sjoerd/Monet-stable/var/monetdb5/dbfarm
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = yes
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	gdk_dbfarm = /ufs/sjoerd/Monet-stable/var/MonetDB
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 31449
# cmdline opt 	mapi_usock = /var/tmp/mtest-17694/.s.monetdb.31449
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbname = mTests_test_BugTracker-2013
# cmdline opt 	mal_listing = 0

# 10:20:45 >  
# 10:20:45 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-17694" "--port=31449"
# 10:20:45 >  


# 10:20:46 >  
# 10:20:46 >  "Done."
# 10:20:46 >  

