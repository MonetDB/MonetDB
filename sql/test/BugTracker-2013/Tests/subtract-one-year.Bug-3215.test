query T rowsort
select cast ('2013-01-01' as date) + interval '-1' year
----
2012-01-01

