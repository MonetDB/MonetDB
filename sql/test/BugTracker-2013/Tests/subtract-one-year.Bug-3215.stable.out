stdout of test 'subtract-one-year.Bug-3215` in directory 'sql/test/BugTracker-2013` itself:


# 10:20:45 >  
# 10:20:45 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "gdk_dbfarm=/ufs/sjoerd/Monet-stable/var/MonetDB" "--set" "mapi_open=true" "--set" "mapi_port=31449" "--set" "mapi_usock=/var/tmp/mtest-17694/.s.monetdb.31449" "--set" "monet_prompt=" "--trace" "--forcemito" "--set" "mal_listing=2" "--dbname=mTests_test_BugTracker-2013" "--set" "mal_listing=0"
# 10:20:45 >  

# MonetDB 5 server v11.13.10 (hg id: 4ee6c3c855fa+)
# This is an unreleased version
# Serving database 'mTests_test_BugTracker-2013', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.629 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://madrid.ins.cwi.nl:31449/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-17694/.s.monetdb.31449
# MonetDB/GIS module loaded
# MonetDB/JAQL module loaded
# MonetDB/SQL module loaded

# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_history.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_compress.sql
# loading sql script: 18_dictionary.sql
# loading sql script: 19_cluster.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 24_zorder.sql
# loading sql script: 25_debug.sql
# loading sql script: 39_analytics.sql
# loading sql script: 40_geom.sql
# loading sql script: 80_udf.sql
# loading sql script: 99_system.sql

# 10:20:45 >  
# 10:20:45 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-17694" "--port=31449"
# 10:20:45 >  

#select cast ('2013-01-01' as date) + interval '-1' year;
% .%2 # table_name
% %2 # name
% date # type
% 10 # length
[ 2012-01-01	]

# 10:20:46 >  
# 10:20:46 >  "Done."
# 10:20:46 >  

